### PR TITLE
Revert to working tritonserver call in notebook using testbooks

### PR DESCRIPTION
**Pull request by jperez999**
Previously we replaced this code because of errors in starting tritonserver based on the tritonserver 22.07 container. Those errors are now resolved and we can run tritonserver from inside the notebook inside the testbooks context again. This is required because of changes in the systems library that now requires the input schema to run the run_ensemble_on_tritonserver function from merlin.systems.triton.utils.

### CODE REVIEW

**Review by benfred — APPROVED** [review]

File: tests/unit/examples/test_building_deploying_multi_stage_RecSys.py [hunk 1]
```diff
@@ -63,20 +60,26 @@ def test_func():
...
-        response = run_ensemble_on_tritonserver(
-            "/tmp/examples/poc_ensemble/", outputs, batch, "ensemble_model"
+        tb2.inject(
+            """
+            import shutil
+            from merlin.core.dispatch import get_lib
+            from merlin.models.loader.tf_utils import configure_tensorflow
+            configure_tensorflow()
+            df_lib = get_lib()
+            batch = df_lib.read_parquet(
```
Comment (rnyak) [inline tests/unit/examples/test_building_deploying_multi_stage_RecSys.py:70]: I am changing the notebooks in this PR  https://github.com/NVIDIA-Merlin/Merlin/pull/618 to return the raw ids. so feeding this batch wont work.  Either we can merge my PR first and then update unit test accordingly, or we can merge this PR, then merge my PR and update unit test again :)

Comment (jperez999) [reply tests/unit/examples/test_building_deploying_multi_stage_RecSys.py:70]: I think we should get what we had working, then you can modify as you see fit. :)

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Pull request merged**